jade-tui

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/bash
+
+# Welcome screen
+gum confirm "$(gum style --border normal --margin '1' --padding '1 2' --border-foreground 212 "$(gum style --foreground 212 '                        mysssym
+                      mysssym
+                    mysssym
+                  mysssym
+                mysssyd
+              mysssyd    N
+            mysssyd    mysym
+          mysssyd      dysssym
+        mysssyd          dysssym
+      mysssyd              dysssym
+      mysssyd              dysssym
+        mysssyd          dysssym
+          mysssyd      dysssym
+            mysym    dysssym
+              N    dysssym
+                 dysssym
+               dysssym
+             dysssym
+           dysssym
+         dysssym')" "" "Welcome to jade-tui" "Ready to make your Crystal installation... ready?")" && CONTINUE=true
+if [[ $CONTINUE != "true" ]]; then
+	echo "Exiting. Have a good day!"
+	exit
+fi
+
+# Select timezone, keymap, locale
+timezone=$(timedatectl list-timezones | gum filter --placeholder "select a timezone")
+keymap=$(localectl list-keymaps | gum filter --placeholder "select a keymap")
+locale=$(cat locales | gum filter --placeholder "select a locale")
```

Comment:
This should be a direct path, something like `/usr/share/jade-tui/locales`, so that it doesn't depend on where the user executes it